ghost/job-manager/lib/JobQueueManager.js

```diff
@@ -133,6 +143,7 @@ class JobQueueManager {
         try {
             await this.pool.exec('executeJob', [jobMetadata.job, jobMetadata.data]);
             await this.jobsRepository.delete(job.id);
+            this.prometheusClient.getMetric('job_manager_queue_job_completion_count').inc({jobName});
```

Comment:
`getMetric()` will return undefined if the metric doesn't exist (e.g. if prometheus isn't enabled), so this should probably be `this.prometheusClient.getMetric(..)?.inc()`